applications/crossbar/doc/port_requests.md

```diff
@@ -265,3 +265,112 @@ All requests are not paginated, with the exception of the `completed` state. Use
      }],
      "status": "success"
     }
+
+
+## Get port request comments
+
+### Request
+
+- Verb: `GET`
+- Url: `/accounts/{{ACCOUNT_ID}}/port_requests/{{PORT_REQUEST_ID}}/comments`
+- Payload: None
+
+
+### Response
+
+```json
+{
+    "data": {
+        "comments": [{
+            "user_id": "{{USER_ID}}",
+            "timestamp": 63607675803,
+            "content": "test",
+            "superduper_comment": false,
+        }, {
+            "user_id": "{{USER_ID}}",
+            "timestamp": 63607678676,
+            "content": "test 2",
+            "superduper_comment": false
+        }, {
+            "user_id": "{{USER_ID}}",
+            "timestamp": 63617678676,
+            "content": "test 3",
+            "superduper_comment": false
+        }, {
+            "user_id": "{{USER_ID}}",
+            "timestamp": 63627678676,
+            "content": "test 5",
+            "superduper_comment": false
+        }]
+    },
+    "status": "success"
+}
+```
+
+
+## Update port request comments
+
+### Request
+
+- Verb: `POST`
```

Comment:
This should really be a PUT to add a comment, and POST/DELETE to `/comments/{COMMENT_ID}` to edit/delete a comment.

You  may consider creating `cb_comments` to handle this (using the preceding `req_noun` to identify what thing the comments are associated with). Then we could add comments to users, devices, PBXes, etc.